tools/swift-test-utils/SwiftTestUtils.swift

```diff
@@ -1,24 +1,24 @@
 import Foundation
 
-struct CommandLineArguments {
-  struct MissingArgumentError: LocalizedError {
+public struct CommandLineArguments {
+  fileprivate struct MissingArgumentError: LocalizedError {
     let argName: String
 
     var errorDescription: String? {
       return "Missing required argument: \(argName)"
     }
   }
-  struct UnkeyedArgumentError: LocalizedError {
+  fileprivate struct UnkeyedArgumentError: LocalizedError {
     let argName: String
 
     var errorDescription: String? {
       return "Unexpectedly found command line argument \(argName) without a key"
     }
   }
 
-  private let args: [String: String]
+  fileprivate let args: [String: String]
```

Comment:
Is there any reason why this an no longer be `private`?

Reply:
Not specifically; keeping it as `private` should be good enough.

Reply:
I always default to `private` because its less verbose and the stricter access level.